public/app/features/migrate-to-cloud/api/index.ts

```diff
@@ -8,50 +8,39 @@ export const cloudMigrationAPI = generatedAPI.enhanceEndpoints({
 
   endpoints: {
     // Cloud-side - create token
-    getCloudMigrationToken(endpoint) {
```

Comment:
There's lots of changes in this file, but all I did was remove `suppressErrorsOnQuery` and rearrange them better.